MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/Models/ListItem.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Web;
+
+namespace MyPerfectOnboarding.Api.Models
+{
+    [DataContract]
```

Comment:
I am still convinced that you should remove [DataContract] and [DataMember] as it should be used with different formatter. JSON.NET can work with it but in the core has its own attributes. It is never good to mix two different approaches and as long as you need to format the output to JSON you should use best practices for JSON formatter.